sdk/python/kfp/cli/compile_.py

```diff
@@ -140,13 +140,58 @@ def parse_parameters(parameters: Optional[str]) -> Dict:
     default=False,
     envvar='KFP_DISABLE_EXECUTION_CACHING_BY_DEFAULT',
     help='Whether to disable execution caching by default.')
+@click.option(
+    '--kubernetes-manifest-format',
+    is_flag=True,
+    default=False,
+    envvar='KFP_KUBERNETES_MANIFEST_FORMAT',
+    help='Whether to output the compiled pipeline version as a Kubernetes manifest.'
+)
+@click.option(
```

Comment:
A lot of these options only affect the Kubernetes manifest generation. Could you please make this clear in the help of each option?